internal/telemetry/client.go

```diff
@@ -43,7 +42,7 @@ type Client interface {
 	// Attr should be called to add additional attributes to this activity.
 	Attr(key, val string)
 	// User returns the user identifier being used by this client
-	User() uuid.UUID
+	User() string
```

Comment:
As noted in the PR description, having `User()` return a string simplifies some test code slightly.

Reply:
Can you expand on exactly how this simplifies the test code?

Reply:
Tests don't need to generate and set a UUID on the mock. The tests can expect a consistent "test-user", and for the install test, that meant I could move the expected output to a `testdata` yaml file
<img width="726" alt="image" src="https://github.com/user-attachments/assets/64b36ee1-9e19-408d-b4e3-0266f83e05be">